### DISCUSSION TITLE
社群运营如何游戏化？

### DISCUSSION THREAD

**Discussion by TianYiFu** [NonceGeek/Move-Camp#92] [Workshop]
时间：2022.06.23 21:00~22:00(Thur.)
分享人昵称：田一夫（Alfred）
内容大纲：
·游戏化的定义
·游戏化的目的
·游戏化思维与传统思维的差异
·为什么我的“游戏”（社群）不好玩
·游戏化实战——八角行为分析法的应用
·游戏化运营实战案例分享

**Comment by leeduckgo** [comment]
🙋‍♂️🙋‍♂️🙋‍♂️

**Reply by Lingyue1213** [reply]
🙋‍♂️🙋‍♂️🙋‍♂️

**Comment by qiwihui** [comment]
🙋‍♂️🙋‍♂️🙋‍♂️

**Reply by 0427282** [reply]
🙋
